src/quo2/components/reacts.cljs

```diff
@@ -0,0 +1,66 @@
+(ns quo2.components.reacts
+  (:require [quo.core :as quo]
+            [quo.react-native :as rn]
+            [quo.theme :as theme]
+            [quo2.foundations.colors :as colors]
+            [status-im.ui.components.icons.icons :as icons]))
+
+(def reaction-styling
+  {:flex-direction :row
+   :justify-content :center
+   :align-items :center
+   :padding-vertical 3
+   :padding-horizontal 8
+   :border-radius 8
+   :gap 4})
```

Comment:
I believe there is no `:gap` property in style can you double check once?

Reply:
Removed, Thanks Sid!